app/javascript/Search/SearchForm.jsx

```diff
@@ -9,19 +22,33 @@ export const SearchForm = forwardRef(
       acceptCharset="UTF-8"
       method="get"
       onSubmit={onSubmitSearch}
+      role="search"
     >
       <input name="utf8" type="hidden" value="✓" />
-      <input
-        ref={ref}
-        className="crayons-header--search-input crayons-textfield"
-        type="text"
-        name="q"
-        placeholder="Search..."
-        autoComplete="off"
-        aria-label="search"
-        onKeyDown={onSearch}
-        value={searchTerm}
-      />
+      <div class="crayons-fields crayons-fields--horizontal">
+        <div class="crayons-field flex-1 relative">
+          <input
+            ref={ref}
+            className="crayons-header--search-input crayons-textfield"
+            type="text"
+            name="q"
+            placeholder="Search..."
+            autoComplete="off"
+            aria-label="search"
+            onKeyDown={onSearch}
```

Comment:
refactor (blocking) - I know it might involve updating some current tests but we should remove this now (and the related `search` function in `Search.jsx` as it now duplicates the submit behaviour 😄